.github/workflows/reusable_dax.yml

```diff
@@ -101,10 +101,28 @@ jobs:
 
       - name: Run the FSDAX tests
         working-directory: ${{env.BUILD_DIR}}
-        run: |
-          UMF_TESTS_FSDAX_PATH=${{env.UMF_TESTS_FSDAX_PATH}} UMF_TESTS_FSDAX_PATH_2=${{env.UMF_TESTS_FSDAX_PATH_2}} ctest -C ${{matrix.build_type}} -R umf-provider_file_memory -V
-          UMF_TESTS_FSDAX_PATH=${{env.UMF_TESTS_FSDAX_PATH}} UMF_TESTS_FSDAX_PATH_2=${{env.UMF_TESTS_FSDAX_PATH_2}} ctest -C ${{matrix.build_type}} -R umf_example_dram_and_fsdax -V
-          UMF_TESTS_FSDAX_PATH=${{env.UMF_TESTS_FSDAX_PATH}} UMF_TESTS_FSDAX_PATH_2=${{env.UMF_TESTS_FSDAX_PATH_2}} ctest -C ${{matrix.build_type}} -R umf-ipc_file_prov_fsdax -V
+        run: >
+          UMF_TESTS_FSDAX_PATH=${{env.UMF_TESTS_FSDAX_PATH}}
+          UMF_TESTS_FSDAX_PATH_2=${{env.UMF_TESTS_FSDAX_PATH_2}}
+          ctest -C ${{matrix.build_type}} -V -R "umf-provider_file_memory|umf_example_dram_and_fsdax|umf-ipc_file_prov_fsdax"
+
+      # TODO: enable the provider_devdax_memory_ipc test when the IPC tests with the proxy library are fixed
+      # see the issue: https://github.com/oneapi-src/unified-memory-framework/issues/864
+      - name: Run the DEVDAX tests with the proxy library
+        working-directory: ${{env.BUILD_DIR}}
+        run: >
+          LD_PRELOAD=./lib/libumf_proxy.so
```

Comment:
I can see in the log: `25: ERROR: ld.so: object './lib/libumf_proxy.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.`

e.g. here: https://github.com/oneapi-src/unified-memory-framework/actions/runs/11685587432/job/32542190787#step:9:31

is this expected...?

Reply:
All is OK, see here:
![image](https://github.com/user-attachments/assets/7ce1d5e0-8ba2-4d4c-af9d-d997f6080726)
https://github.com/oneapi-src/unified-memory-framework/actions/runs/11699101843/job/32580506795

This error message comes from `ctest` when `libumf_proxy.so` has already been loaded (`proxy_lib_create_common: proxy_lib initialized`). I do not know why ctest tries to load it again.